### PR TITLE
extra_param parameter in send request.

### PR DESCRIPTION
**Pull request by sreeteja06**
extra_param option added in send request, which is used for setting variables that are set in  SendOTP template.

### CODE REVIEW

File: src/msg91OTP.js [hunk 1]
```diff
@@ -55,7 +55,8 @@ class Msg91Otp {
         otp_expiry: args.expiry || this.defaultOtpExpiry,
         otp_length: args.otpLength || this.defaultOtpLength,
         email: args.email || '',
-        invisible: args.invisible || 1
+        invisible: args.invisible || 1,
+        extra_param: args.extra_param || ''
```
Comment (sumeetkbhardwaj) [inline src/msg91OTP.js:59]: @sreeteja06 is this backward compatible with API version less than v5?

Comment (sreeteja06) [reply src/msg91OTP.js:59]: @sumitsk20  yes works with v1, v2, v3, v4, and v5.

**Comment by sumeetkbhardwaj** [comment]
@sreeteja06 can you also update the version in the package.json from 2.0.1 to 2.0.2 in your pr and update it?

**Comment by sreeteja06** [comment]
@sumitsk20 updated package.json

**Comment by sreeteja06** [comment]
I haver added types to the package, one verify at ur end and merge the pull request. @sumitsk20

**Pull request merged**